src/GslibIO.jl

```diff
@@ -155,4 +196,58 @@ function save(file::File{format"GSLIB"}, sdata::AbstractData)
        propnames=vars)
 end
 
+# low level function for saving `data` to a legacy GSLIB format using `varnames` as variable names
+function save_legacy(filename::AbstractString, data::AbstractMatrix, varnames::NTuple, header::AbstractString, na)
+  @assert size(data, 2) == length(varnames) "Invalid data for the specified variable names"
+  nvars = size(data, 2)
+
+  open(filename, "w") do f
+    write(f, "$header\n")
+
+    write(f, "$nvars\n")
+    for v in varnames
+      write(f, "$v\n")
+    end
+
+    # handle missing values
+    replace!(data, NaN=>na)
+
+    writedlm(f, data, ' ')
+  end
+end
+
+"""
+    save_legacy(file, sdata)
+
+Save spatial data `sdata` to `filename` using standard GSLIB format. It replaces NaNs with `na` values
+and it uses `coordnames` as the given variable name to coordinates
+"""
+function save_legacy(filename::AbstractString, sdata::AbstractData; 
+                     coordnames=(:x, :y, :z), na=-999.0, header="# This file was generated with GslibIO.jl")
+  table = values(sdata)
+  sdomain = domain(sdata)
+  
+  if sdomain isa PointSet
+    # add `coordinates` to data and `coordnames` to `varnames`
```

Comment:
```suggestion
    # add coordinates to data and coordnames to varnames
```
No need to use backquotes in comments. They are only useful in docstrings and markdown previews.